include/RevCore.h

```diff
@@ -54,20 +53,22 @@
 #include "../common/include/RevCommon.h"
 #include "../common/syscalls/syscalls.h"
 
+#include "AllRevInstTables.h"
+
 namespace SST::RevCPU {
 class RevCoProc;
 
 class RevCore {
 public:
   /// RevCore: standard constructor
   RevCore(
-    unsigned                  Id,
-    RevOpts*                  Opts,
-    unsigned                  NumHarts,
-    RevMem*                   Mem,
-    RevLoader*                Loader,
+    unsigned                  id,
+    RevOpts*                  opts,
+    unsigned                  numHarts,
+    RevMem*                   mem,
+    RevLoader*                loader,
     std::function<uint32_t()> GetNewThreadID,
-    SST::Output*              Output
+    SST::Output*              output
```

Comment:
It is not necessary to use alternative case between constructor parameters and class members. During member initialization, the class member is distinct from constructor parameters, so `id( id ), opts( opts ), numHarts( numHarts )` etc. can be used and the member is chosen on the outside of the initializer, and the constructor parameter is chosen on the inside of the initializer.